logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultChronoZonedDateTimeAssertions.kt

```diff
@@ -44,4 +47,86 @@ class DefaultChronoZonedDateTimeAssertions : ChronoZonedDateTimeAssertions {
     ): Assertion = container.createDescriptiveAssertion(IS_EQUAL_TO, expected) {
         it.isEqual(expected)
     }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isBefore(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isBefore(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isBeforeOrEqual(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isBeforeOrEqual(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isAfter(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isAfter(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isAfterOrEqual(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isAfterOrEqual(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isEqual(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isEqual(parseZonedDateTime(expected))
+    }
+
+    private fun parseZonedDateTime(data: String): ZonedDateTime {
+        /**
+         * The specification of the zone can look like this [Europe/Paris]
+         * and needs to be handled by the native jdk implementation.
+         */
+        if (data.indexOf('[') > -1 && (data.indexOf('[') < data.indexOf(']'))) {
```

Comment:
personally I would assign `data.indexOf('[') `  to a variable instead of calling indexOf twice but it's also OK this way

Reply:
I am not sure if we really should support [Europe/Paris] and the like. In the issue description I had:
``` 
Right now we don't intend to support things like Europe/Zurich as ZoneDesignators (would be supported by jdk). So do a check upfront if in correct format and throw a DateTimeParseException if wrong and only then use ZonedDateTime.parse
```
Now that you have implemented it, I would say keep it. But state a note in KDoc that its supported only experimentally and could be removed at any time.

Reply:
Alright, I am dropping it for now.